public/static/docs/command-reference/get.md

```diff
@@ -65,6 +65,13 @@ name.
   the file or directory from. The tip of the default branch is used by default
   when this option is not specified.
 
+- `--show-url` - Returns path or url to the location in remote for given path,
+  from where the **file** will be downloaded. `path` is used to specify the location
+  of the target artifact within the source project in `url`, relative to the project's
+  root. `url` can be an address to a DVC project or a path to a local DVC project.
+  When this option is specified, it will not download the files.
+  Git-only repository is not supported.
```

Comment:
Probably, non-DVC repository would have been better?

Reply:
let's not explain the positional args (`path` and `url` here) - they have the same meaning and are probably already described, right?

Reply:
and/or even put a link to the example below ...

Reply:
Yep, `url` and `path` are explained in the description. No need to do it again here, you can simply reference them. You can even use ``(`url`/`path`)`` as in the introductory paragraph.

Reply:
Oh and
> Probably, "non-DVC repository"...

Yes, I like that term better. But typically we begin with applicability/restriction sentences in option paragraphs, for example see the language used here:
![image](https://user-images.githubusercontent.com/1477535/72761955-6517fb80-3ba3-11ea-9032-aaba43fda741.png)

Thanks

Reply:
Updated in 072defa3.